### DISCUSSION TITLE
21.5.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1350] [General]
## Releases Notes for 21.5.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/draw.io-21.5.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/draw.io-21.5.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/draw.io-universal-21.5.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/drawio-amd64-21.5.0.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/drawio-x86_64-21.5.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.5.0/drawio-x86_64-21.5.0.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Implements a number of security hardening measures designed to reduce the chances of a exploit chain.
- Updates to [draw.io core 21.5.0](https://github.com/jgraph/drawio/blob/v21.5.0/ChangeLog). All changes from 21.4.1 to 21.5.0 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.5.0'>21.5.0</a>.</em>

**Comment by FredSeb** [comment]
Hello
I need to know how in the installation , I suppress Update menu in Draw.io.
Think you for your help.
